plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -135,16 +154,82 @@ public ResultSet getTables(Connection connection, Optional<String> schemaName, O
     @Override
     public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connection connection, JdbcTypeHandle typeHandle)
     {
+        Optional<ColumnMapping> mapping = getForcedMappingToVarchar(typeHandle);
+        if (mapping.isPresent()) {
+            return mapping;
+        }
+
+        // TODO (https://github.com/trinodb/trino/issues/9215) Implement proper type mapping and add test
```

Comment:
Maybe a link to https://github.com/trinodb/trino/issues/9215 here would be helpful - it mentions a few possible ways to ingest data into Druid (similar to how we test Pinot today).

Reply:
Existing TODO comment already has a link to #9215?

Reply:
I'm going blind apparently. 😛